src/platform/packages/shared/kbn-field-formats-common/constants.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const EMPTY_LABEL = i18n.translate('fieldFormats.blankLabel', {
+  defaultMessage: '(blank)',
+  description: 'Represents the label used to replace an empty string value in tables and charts',
+});
+
+export const NULL_LABEL = i18n.translate('fieldFormats.nullLabel', {
+  defaultMessage: '(null)',
+  description: 'Represents the label used to replace an null value in tables and charts',
+});
+
+export const NAN_LABEL = 'NaN';
```

Comment:
The PR description says 

> for NaN numbers -> use the term N/A

just to clarify, it makes more sense like it's here in the code, is just the PR description outdated?

Reply:
PR description outdated, updated right now